client/src/Styles/Store.css

```diff
@@ -0,0 +1,214 @@
+.store {
+    background-color: #393939;
+    position: fixed;
+    height: 100vh;
+    width: 100%;
+    top: 0;
+    left: 0;
+    display: flex;
+}
+
+.image-display {
+    width: 100%;
+    height: auto;
+}
+
+.item-icon-coin {
+    width: 40px;
+    height: 40px
+}
+
+.currency {
+    position: absolute;
+    border-radius: 20px;
+    border: 5px solid #248C28;
+    background: #4CAF50;
+    color: #FFF;
+    text-align: center;
+    font-size: 48px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: normal;
+    flex-shrink: 0;
+    display: flex;
+    align-items: center;
+    gap: 15px;
+    padding: 10px;
+    width: 250px;
+    height: 60px;
+    right: 0;
+    top: 25px;
+}
+
+.menu {
+    background-color: #F0F0F0;
+    display: flex;
+    flex-direction: column; 
+    flex-shrink: 0;
+    align-items: center;
+    width: 450px;
+    height: 100vh;
+}
+
+.menu-title {
+    color: #036;
+    text-align: right;
+    font-size: 80px;
+    font-style: normal;
+    font-weight: bold;
+    line-height: normal;
+    margin-top: 25px;
+    margin-bottom: 50px
+}
+
+.menu-item-buttons {
+    display: flex;
+    flex-direction: column;
+    row-gap: 15px;
+}
+
+.menu-exit {
+    margin-top: auto;
+    margin-bottom: 25px;
+}
+
+.item-grid {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 172px;
+    margin-left: auto;
+    margin-right: auto;
+    justify-content: center;
+    overflow: scroll;
```

Comment:
I think changing this to `auto` instead of scroll would be better. For my screen its giving a weird padding on the right and bottom where the scroll bar is supposed to be in even when there's no scroll.

Reply:
Changed it to auto